.zenodo.json

```diff
@@ -68,6 +68,9 @@
       "name": "Lee, Gregory R.",
       "orcid": "0000-0001-8895-2740"
     },
+    {
+      "name": "Wang, Hao-Ting"
```

Comment:
@htwangtw Please let me know if you would like your name presented differently, and if you have an affiliation/ORCID that you want associated with your contributions.

This will be how your name is presented as an author on the [Zenodo archive](https://doi.org/10.5281/zenodo.591597). If you prefer not to be an author, let me know.

Reply:
Please add: `"orcid": "0000-0003-4078-2038"`
Thanks so much!